src/components/card/contactInfoCard/ContactInfoCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useTheme } from "../../themeProvider/ThemeProvider";
+
+const ContactInfoCard = () => {
+  const darkMode = useTheme();
+
+  return (
+    <div className={`ui fluid link card  ${darkMode ? "inverted" : ""}`}>
+      <div className="ui basic segment">
+        <div className="ui image medium centered">
+          <img
+            alt="avatar"
+            src={`https://avatars.dicebear.com/api/bottts/${
+              Math.random() * (100000 - 1) + 1
+            }.svg`}
```

Comment:
extract this logic in a util, check comment on Tanvir's PR

Reply:
okay @faran4engg